source/gui/guiHelper.py

```diff
@@ -167,6 +174,21 @@ def control(self):
 	def sizer(self):
 		return self._sizer
 
+class LabeledTextCtrl(LabeledControlHelper):

```

Comment:
Could you give this a doc string that clarifies why this subclass has been created?

Reply:
Yes, I'll do this. Trying to achieve this small variation has made me think carefully about the implementation here. I think we should start to move this towards a more component driven implementation. The initial goal was to try to streamline the GUI code, making it simpler (and less error prone) to do the common things, I think this has been successful. Though now I would like to make this a little more flexible too.

Reply:
I decided to remove this class, trying to find a way to implement this variation in a clean way has inspired me to take another approach. For now, this means that two textCtrls that should grow with the dialog will not. I don't think this PR should be blocked on this.